src/content/docs/en/recipes/dynamically-importing-images.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Dynamically Import Images
+description: Learn how to dynamically import images using Vite's import.meta.glob function
+i18nReady: true
+type: recipe
+---
+import FileTree from '~/components/FileTree.astro'
+
+When working with [images](/en/guides/images/), there will be times where you will need to dynamically import the image paths. 
```

Comment:
```suggestion
Local [images](/en/guides/images/) must be imported into `.astro` files in order to display them. There will be times where you will want or need to dynamically import the image paths of your images instead of explicitly importing each individual image.
```
I like emphasizing that images must be imported here, because I feel like that's something we keep reminding people of!  :smile:  This also gives a bit of context re: what problem this solves: avoiding needing to manually and individually import every image you might want.